app/controllers/application_controller.rb

```diff
@@ -18,13 +18,7 @@ class ApplicationController < ActionController::Base
 
 
   rescue_from "AccessGranted::AccessDenied" do |exception|
-    redirect_path = if current_user.present?
-      root_path
-    else
-      new_user_session_path
-    end
-
-    redirect_to redirect_path, alert: "You don't have permission to access requested page."
+    redirect_to root_path, alert: "You don't have permission to access that page."
   end
```

Comment:
Before, If they tried to access a page when they weren't logged in that would require a login, it would redirect them to a login screen. 

I think this was useful. The idea was you might bookmark a partiuclar admin page you need to use frequenty or whatever. 

I guess it was changed here because we can't actually easily redirect to login anymore?  I guess we could have a landing page with a login button on it (`/login` could go there too), but I guess it's not _that_ useful unless after logging in you actually return to where you were trying to go in the first place, which is kind of a pain to keep track of through a process that stops on a "login button landing page". 

I dunno, what do you think, worth thinking about, or it's fine like above?  Ask Gabriela?

Reply:
We did discuss this today briefly and decided to leave this as is and make changes, if any, in separate PRs after this has been in production for a bit.